homeassistant/components/mqtt/switch.py

```diff
@@ -51,6 +51,8 @@
 CONF_STATE_ON = "state_on"
 CONF_STATE_OFF = "state_off"
 
+PAYLOAD_NONE = "None"
```

Comment:
Should this be configurable?

Reply:
I do not want to suggest to make this configurable. It should be compatible with a `None` value that comes from a json `null` value. From a template this is a string. I'd like to move PAYLOAD_NONE to .const. I assume this ia already in `dev` with the approved binary_sensor integration.
If you still think this should be configurable, than we need to do this for all platforms that get this support.